Bottom: 49fb0d8121f169fb2458ae7eb077a72f3aebd5f7
Top:    a296bf6554213d743d415059c54d374861431ea3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 00:00:27 +0100

Refresh of hid-gerber-don-t-use-the-core-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index ec32be5..2c9f88f 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -529,7 +529,7 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
         DrawLayerGroup (group, drawn_area);
     }
 
-  CountHoles (drawn_area, &nplated, &nunplated);
+  CountHoles (&nplated, &nunplated, drawn_area);
 
   if (nplated && set_layer ("plated-drill", SL (PDRILL, 0)))
     DrawHoles (true, drawn_area);
